examples/asr/asr_adapters/scoring_and_analysis.py

```diff
@@ -0,0 +1,339 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This script is used to analyze the results of the experiments from a CSV file.
+
+Usage:
+    python scoring_and_analysis.py \
+        --csv <path to cleaned result csv file> \
+        --dataset_type_column <column in csv with the dataset types>
```

Comment:
Move the basic usage documentation inside this script.

Reply:
Done